tracer/src/Datadog.Trace/RemoteConfigurationManagement/Product.cs

```diff
@@ -22,6 +21,8 @@ protected Product()
 
         public event EventHandler<ProductConfigChangedEventArgs> ConfigChanged;
 
+        public event EventHandler<ProductConfigChangedEventArgs> ConfigRemoved;
```

Comment:
The current API here is a bit confusing - there is `ConfigChanged` and `ConfigRemoved`, but no `ConfigAdded`. 
Should we rename `ConfigChanged` to `ConfigAddedOrChanged`, or otherwise aim towards having 3 separate events?
Note that right now for Dynamic Instrumentation probes are immutable so changing a probe means deleting it and creating a new one in its place, but we aim to change this so that any change will merely update the data and increment the Remote Configuration record's `version` attribute.
to a probe will

Reply:
I prefer not to change it right now because we are not the only product using this API. This change should be consolidated with other products before.
I think we should do this, when we are ready to support `update` events explicitly.